app/services/reports/bulk_export_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Reports
+  class BulkExportService < ::WasteExemptionsEngine::BaseService
+    def run(*)
+      first_day_of_the_month = starts_from
+
+      while first_day_of_the_month < Date.today
```

Comment:
Like my previous comment, it would be great if we could ensure that any exceptions thrown during the generation of any one file does not cause the service to stop generating the others.

It comes to mind here, but there may be better places to do this.